package.json

```diff
@@ -115,6 +115,12 @@
   "jspm": {
     "format": "amd"
   },
+  "eslintConfig": {
```

Comment:
we've been using `.eslintrc` files which I think give a bit more granularity.  So there could be one at the root with 
```
{env: {node: true}} // does this need browser as well?
```
and then in the test folder, you can do
```
{env: {mocha: true}) // only, since .eslintrc file are additive
```

Reply:
Sure. I'm not quite familiar with those files, let me try the suggestion above.

Reply:
Added